examples/run_action_recognition.py

```diff
@@ -65,7 +65,7 @@
     # Create a logistic regression classifier
     gesture_classifier = LogisticRegression(num_in=backbone_network.feature_dim,
                                             num_out=30)
-    gesture_classifier.load_state_dict(weights['gesture_recognition'])
+    gesture_classifier.load_state_dict(weights['action_recognition'])
```

Comment:
Should these variables also be renamed to `action_classifier`?

Reply:
Yep, good point